src/vos/evtree.c

```diff
@@ -185,7 +185,8 @@ evt_weight_diff(struct evt_weight *wt1, struct evt_weight *wt2,
 {
 	/* NB: they can be negative */
 	wt_diff->wt_major = wt1->wt_major - wt2->wt_major;
-	wt_diff->wt_minor = wt1->wt_minor - wt2->wt_minor;
+	/** wt2 is the difference to the original mbr */
```

Comment:
Minor: this function is just a common function comparing two weights, this comment should be put somewhere else.

Reply:
fixed